crates/shared/src/baseline_solver.rs

```diff
@@ -61,10 +61,10 @@ pub fn estimate_buy_amount<'a, L: BaselineSolvable>(
     let sell_token = path.first()?;
     path.iter()
         .skip(1)
-        .fold(
-            Some((sell_amount, *sell_token, Vec::new())),
+        .try_fold(
```

Comment:
oops! 🙈

Reply:
Still don't understand why git is not able to update a PR diff correctly when the PR is based off of a different branch that then got merged to `main`.
Merged `main` into this branch and now things are correct again.